lib/compat/wordpress-6.9/class-gutenberg-rest-old-templates-controller.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-class Gutenberg_REST_Old_Templates_Controller extends WP_REST_Templates_Controller {
```

Comment:
I guess this is required as well? Probably should be moved to `compat/wordpress-7.0` if so.